### PR TITLE
Add gl-vertex-attrib-i-render.html for WebGL 2.0

### PR DESCRIPTION
**Pull request by legendlee1314**

### CODE REVIEW

**Comment by legendlee1314** [comment]
@kenrussell , PTAL! Thanks!

This is copied from `conformance/attrib/gl-vertex-attrib-render.html`. However, it has the minimum modification for testing `vertexAttribIPointer` and `vertexAttribI4[ui][v]`.

File: sdk/tests/conformance2/attribs/gl-vertex-attrib-i-render.html [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+              gl[tests[ii]](1, vals[ii][0], vals[ii][1], vals[ii][2], vals[ii][3]);
+            } else {
+              gl[tests[ii]](1, vals[ii]);
+            }
+            gl.drawArrays(gl.TRIANGLE_STRIP, 0, 4);
+
+            if (checkRedPortion(gl, 50, 50 * 0.7, 50 * 0.8)) {
+              testPassed('Attribute of ' + tests[ii] + ' was set correctly');
+            } else {
+              testPassed('Attribute of ' + tests[ii] + ' was not set correctly');
```
Comment (kenrussell) [inline sdk/tests/conformance2/attribs/gl-vertex-attrib-i-render.html]: Should be testFailed.

**Comment by kenrussell** [comment]
Thanks for this contribution. It looks good overall. Please fix the bug above, squash commits, re-upload this pull request, and comment here when done.

**Comment by legendlee1314** [comment]
@kenrussell , thank you! I have addressed your comment. Please review again!

**Comment by kenrussell** [comment]
Thanks, looks good.

**Pull request merged**